Model/Player.py

```diff
@@ -0,0 +1,26 @@
+from Server.Database.DBMethods import DBMethods
+
+
+class Player:
+    def __init__(self, player_id, username, password):
+        self.id = player_id
+        self.username = username
+        self.password = password
+
+    def to_dict(self):
+        return self.__dict__
+
+    @staticmethod
+    def create_object_from_details_in_db(player_username):
```

Comment:
create is a keyword that hints a creation for a new thing, in your case is more fetching
create_object_from_details_in_db -> fetch_player_from_db